core/server/config/index.js

```diff
@@ -179,8 +192,14 @@ ConfigManager.prototype.set = function (config) {
             availableApps:    this._config.paths.availableApps || {},
             clientAssets:     path.join(corePath, '/built/assets/')
         },
+        scheduling: {
+            postScheduling: {
```

Comment:
The intention here is to have one small moving piece that handles making tasks happen at a correct time, which can be completely agnostic of what those tasks are. 

We can and will then build out many, many features on top of that piece. The configuration should not need updating each time we add a new scheduling feature and my expectation is that each blog would only ever have one scheduling adapter installed. 

All of the fancy features would live in the tools that are leveraging the adapter, not in the adapter itself, and so there would never be any need to have more than one.